groovy-language-server/src/main/java/com/palantir/ls/groovy/GroovycWrapper.java

```diff
@@ -355,18 +353,18 @@ private void resetCompilationUnit() {
     }
 
     private void parseAllSymbols() {
-        Map<String, Set<SymbolInformation>> newFileSymbols = Maps.newHashMap();
+        Map<Path, Set<SymbolInformation>> newFileSymbols = Maps.newHashMap();
```

Comment:
why is the key here a path not uri? haven't finished reading all your code yet but won't that save some conversion time?